### ISSUE TITLE
report() on iOS crash

### ISSUE THREAD

**Issue by mswokk** [ianlin/react-native-firebase-crash-report#14]
In RNFirebaseCrashReport.m

RCT_EXPORT_METHOD(report:(NSString *)message)
{
    FIRCrashLog(message);
    assert(NO);  // ===> Crash. 
}

Is it intented?

**Comment by ianlin** [comment]
Yeah it unfortunately is as stated in Firebase's [iOS Document][1] there is no such thing like [Android's FirebaseCrash.report()][2].

[1]: https://firebase.google.com/docs/crash/ios#create_your_first_error
[2]: https://firebase.google.com/docs/crash/android#create_your_first_error
